mods/d2k/rules/structures.yaml

```diff
@@ -593,7 +593,7 @@ large_gun_turret:
 		QuantizedFacings: 32
 	Armament:
 		Weapon: TowerMissile
-		LocalOffset: 256,384,768, 256,-384,768
+		LocalOffset: 256,0,768
```

Comment:
Alternating would be great, as long as we don't change the effective DPS of the unit (and don't make it fire 2 missiles at once).